dev_apps/music-nga/elements/music-artwork.js

```diff
@@ -250,6 +278,10 @@ Object.defineProperty(proto, 'repeat', {
   set: function(value) {
     value = REPEAT_VALUES.indexOf(value) !== -1 ? value : REPEAT_VALUES[0];
     this.setAttribute('repeat', value);
+
+    this.els.repeat.dataset.l10nId = 'repeat-' + value;
+
+    this.onDOMLocalized();
```

Comment:
You could inline the `onDOMLocalized` function here:

    document.l10n.ready.then(() => document.l10n.translateFragment(shadowRoot));